providers/aws/services/securityhub/securityhub_service.py

```diff
@@ -0,0 +1,91 @@
+import threading
+from dataclasses import dataclass
+
+from lib.logger import logger
+from providers.aws.aws_provider import generate_regional_clients
+
+
+################## SecurityHub
+class SecurityHub:
+    def __init__(self, audit_info):
+        self.service = "securityhub"
+        self.session = audit_info.audit_session
+        self.audited_account = audit_info.audited_account
+        self.regional_clients = generate_regional_clients(self.service, audit_info)
+        self.securityhubs = []
+        self.__threading_call__(self.__describe_hub__)
+
+    def __get_session__(self):
+        return self.session
+
+    def __threading_call__(self, call):
+        threads = []
+        for regional_client in self.regional_clients.values():
+            threads.append(threading.Thread(target=call, args=(regional_client,)))
+        for t in threads:
+            t.start()
+        for t in threads:
+            t.join()
+
+    def __describe_hub__(self, regional_client):
+        logger.info("SecurityHub - Describing Hub...")
+        try:
+            get_enabled_standards_paginator = regional_client.get_paginator(
+                "get_enabled_standards"
+            )
+            standards = ""
+            for page in get_enabled_standards_paginator.paginate():
+                for standard in page["StandardsSubscriptions"]:
```

Comment:
Is it possible to have two functions? One for the `get_enabled_standards` and other for `describe_hub`?

Reply:
Get enabled standards is done to check if there is a hub in order to do the describe_hub operation

Reply:
Yes, I get it but would it make sense to split it into two different functions? I prefer to do it so.

Reply:
I believe that it is not possible since one function depends regionally on the other...